lib/buildsystems/autotools/v1.rb

```diff
@@ -0,0 +1,43 @@
+require 'package'
+
+class Autotools_v1 < Package
+  @_buildsystems_autotools_version = '1.0.0'
+
+  def self.configure_options (options = '')
+    if options
+      @configure_options = options
+    end
+    return @configure_options
+  end
+
+  def self.check? (bool = false)
+    @check = bool
+    return @check
+  end
+
+  def self.build
+    # Run autoreconf if necessary
+    unless File.executable? './configure'
+      if File.executable? './autogen.sh'
+        system 'NOCONFIGURE=1 ./autogen.sh'
```

Comment:
`NOCONFIGURE` is an environment variable mostly accepted by freedesktop projects, it makes sense to have it just in case but `--no-configure` is more commonly used.

Reply:
`--no-configure` is a parameter and should be placed after `./autogen.sh`

Reply:
see [e5ce5a1](https://github.com/chromebrew/chromebrew/pull/6306/commits/e5ce5a12cfa4f6a0720bf3040087a1855b00baa4)